jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  transform: {
+    '\\.(ts|tsx)$': 'ts-jest',
+  },
+  testRegex: '/__tests__/.*\\.(ts|tsx|js)$',
```

Comment:
I don't think this is consistent with our other repos, but it's fine for now. We should discuss folder architecture wrt tests (not super important, but probably easier to figure out now than migrate stuff later)

Reply:
Yeah, as long as this follows general open source conventions, I'm not as concerned with it being consistent with other internal Heap repos.  That's the default for jest, so I'd suggest we stick with it.